src/common/Utils.ts

```diff
@@ -123,3 +123,12 @@ export function isCustomTelemetryEnabled():boolean {
         .get(CUSTOM_TELEMETRY_FOR_POWER_PAGES_SETTING_NAME);
     return isCustomTelemetryEnabled as boolean;
 }
+
+export function getUserAgent(): string {
+    const userAgent = "{product} / {product-version} {comment}";
+
+    return userAgent
+        .replace("{product}", EXTENSION_NAME)
+        .replace("{product-version}", getExtensionVersion())
+        .replace("{comment}", getExtensionType()+'; '+getExtensionVersion());
```

Comment:
Looks like the extension version is there twice. Remove from the comment?

Reply:
removed